src/cargo/core/features.rs

```diff
@@ -776,6 +776,7 @@ unstable_cli_options!(
     no_index_update: bool = ("Do not update the registry index even if the cache is outdated"),
     package_workspace: bool = ("Handle intra-workspace dependencies when packaging"),
     panic_abort_tests: bool = ("Enable support to run tests with -Cpanic=abort"),
+    path_bases: bool = ("Allow paths that resolve relatively to a base specified in the config"),
```

Comment:
Is there a reason you did this as a `-Z` instead of as `cargo-features`?

`-Z` with `Cargo.toml` unstable features is usually reserved for "we gracefully degrade when we ignore these parts of the manifest".

Reply:
Ah, didn't realize that was a thing (sorry, I inherited this from Jonhoo a while ago...)

Reply:
In case you didn't find it, the docs for this is at https://doc.rust-lang.org/nightly/nightly-rustc/cargo/core/features/index.html

Reply:
ok